include/sys/zfs_stat.h

```diff
@@ -44,6 +44,8 @@ typedef struct zfs_stat {
 	uint64_t	zs_mode;
 	uint64_t	zs_links;
 	uint64_t	zs_ctime[2];
+	uint64_t	zs_data_block_size;
+	uint64_t	zs_indirect_block_size;
```

Comment:
While not needed by this patch I think it would be useful to add `zs_dnode_size` here as well.